datamonster_api/lib/company.py

```diff
@@ -40,23 +41,24 @@ def __eq__(self, obj):
     @property
     def pk(self):
         """
-        :return: (int) the primary key (pk)
+        :return: (int) the unique internal identifier for the company
         """
         return int(self.id)
 
     @property
     def datasources(self):
         """
-        :return: (iter) iterable of Datasource objects associated with this company, memoized
+        :return: (iter) iterable of ``Datasource`` objects associated with this company to which the user has access, memoized
         """
         if not hasattr(self, "_datasources"):
             self._datasources = self.dm.get_datasources(company=self)
 
         return self._datasources
 
     def get_details(self):
-        """Get details (metadata) for this company
+        """Get details (metadata) for this company with keys:
+            ``id``, ``name``, ``quarters``, ``ticker``, and ``type`` (always company)
```

Comment:
what is "always company"? - also when get_details changes (the whole purpose of how this was created is that changes are made on the server side and nothing needs to change here), this will no longer be relevant. this description should be more generic